internal/pkg/plugin/harbor/harbor.go

```diff
@@ -1 +1,9 @@
 package harbor
+
+var defaultDeploymentList = []string{
+	"habor-chartmuseum",
+	"harbor-core",
+	"harbor-jobserice",
```

Comment:
It's typo?

Reply:
![image](https://user-images.githubusercontent.com/61452000/179936528-550a7e09-344c-440a-a2f3-21609e3ee8ae.png)

Reply:
Maybe they are `harbor-chartmuseum` and `harbor-jobservice`